tasks/task_database_cue.c

```diff
@@ -19,6 +19,12 @@
 #include <ctype.h>
 #include <string.h>
 
+// regex
+#include <iostream>
+#include <string>
+#include <regex>
+#include <iterator>
```

Comment:
Don't think these includes are needed.

Reply:
Indeed, please don't use any C++ headers in C code.

Reply:
I fixed that, and no more c++ code, just C, it compiles with  make C89_BUILD=1.  I've tested the scanner with Robs new rdb files and its working.